src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -63,6 +63,14 @@ private ScheduleView(Category category, string name, ScheduleType type)
             SafeInit(() => InitScheduleView(category, name, type));
         }
 
+        /// <summary>
+        /// Private constructor
```

Comment:
can you note what this constructor is for vs the others? This one seems very specific.